statsmodels/genmod/generalized_linear_model.py

```diff
@@ -1309,7 +1309,7 @@ def resid_pearson(self):
     @cache_readonly
     def resid_working(self):
         # Isn't self.resid_response is already adjusted by _n_trials?
-        val = (self.resid_response / self.family.link.deriv(self.mu))
+        val = (self.resid_response * self.family.link.deriv(self.mu))
         val *= self._n_trials
```

Comment:
I assume this is only for Binomial. I didn't look into it. 
GLMBinomial has a unit test, that passes

The main problem I had in getting the test to run was with offset or exposure because I had made a mistake in rewriting the tests, but that also seems to work fine. There are some problems hiding in the test suite compared to Stata because some of the individual required test precisions are quite low. But that's independent of correcting the working residuals.